staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -116,7 +116,9 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringSliceVar(&s.EtcdServersOverrides, "etcd-servers-overrides", s.EtcdServersOverrides, ""+
 		"Per-resource etcd servers overrides, comma separated. The individual override "+
-		"format: group/resource#servers, where servers are URLs, semicolon separated.")
+		"format: group/resource#servers, where servers are URLs, semicolon separated. "+
+		"Note that this applies only to resources compiled into this server binary, "+
+		"so it does not affect resources served by aggregated API servers nor ones defined by CRDs.")
```

Comment:
hmm... this help text is used for any API server built on top of the k8s.io/apiserver library, which can include aggregated API servers. The important bit is that the flag only applies to the resources compiled into the particular server binary being run... that's what I meant for the help text to be limited to.

Including help text that references CRDs in a library used for non-kubeapiserver binaries is confusing.

Reply:
I see your point and it makes sense to me, I will fix it. That said, I still think users should be aware of what I was trying to say here, so I will look for a more proper location in the documentation. Do you have any suggestion?